ui/tabs.js

```diff
@@ -471,10 +477,7 @@ return $.widget( "ui.tabs", {
 	},
 
 	_createPanel: function( id ) {
-		return $( "<div>" )
-			.attr( "id", id )
-			.addClass( "ui-tabs-panel ui-widget-content ui-corner-bottom" )
-			.data( "ui-tabs-destroy", true );
+		return $( "<div>" ).attr( "id", id ).data( "ui-tabs-destroy", true );
```

Comment:
Looks like the `.addClass()` call was duplicating another one and you therefore dropped it?